Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -2092,6 +2121,51 @@ def get_mapping_fields(client: QRadarClient) -> dict:
     return fields
 
 
+def get_custom_properties(
+        client: QRadarClient, limit: Optional[str] = None, field_name: Optional[str] = None,
+        like_name: Optional[str] = None, filter: Optional[str] = None, fields: Optional[str] = None) -> dict:
```

Comment:
```suggestion
        like_name: Optional[str] = None, filter_: Optional[str] = None, fields: Optional[str] = None) -> dict:
```

Reply:
but it'll override the python's filter function...

Reply:
still bad practice, but since the whole code in this integration is built wrong just keep it as is